Jenkinsfile

```diff
@@ -45,7 +45,7 @@ def sanitized_JOB_NAME = JOB_NAME.toLowerCase().replaceAll('/', '-').replaceAll(
 
 def el7_component_repos = ""
 def sle12_component_repos = ""
-def component_repos = ""
+def component_repos = "spdk@PR-9 dpdk@PR-6"
```

Comment:
So, we cannot land with these still in here.  These component PRs need to land first, then remove the above PRs from this file.

However!  If the landing of that updated spdk and/or dpdk RPM PRs would break current DAOS, you need to push a patch to DAOS restricting it to using the max. compatible versions, which can just be the current versions we are using.  You do that with `Requires:` in the [daos.spec](https://github.com/daos-stack/daos/blob/12c119af2b83143e59bbe35ac28d57488b374ee7/utils/rpms/daos.spec).  There is in fact already a:
https://github.com/daos-stack/daos/blob/12c119af2b83143e59bbe35ac28d57488b374ee7/utils/rpms/daos.spec#L85
which you can just update with a ` <= <currently_used_version>`.

Reply:
https://github.com/daos-stack/daos/pull/1374